systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -15,6 +15,16 @@
 
 ## User Interfaces
 
+![User Interfaces Building Block Diagram](./diagrams/05-user-interfaces.drawio.svg)
+
+| Building Block          | Type/Context   | Responsibility                                                              |
+| ----------------------- | -------------- | --------------------------------------------------------------------------- |
+| Pages & Content         | MediWiki       | Provide entity data, mainly through wiki articles (web UI)                  |
```

Comment:
Claiming that the responsibility of a Client's pages & content is to provide entity data doesn't sound quite right to me. This nicely ties in with the missing link from the "Lua and Parser Functions" row in the [Client's technical context](https://wmde.github.io/wikidata-wikibase-architecture/systems/WikibaseClient/03-Context_and_Scope.html#technical-context) table. Entity data is loaded on wiki pages *only* through Lua and Parser Functions so this would fill in the missing link.

I think we could either mention how pages and content relate to Lua and Parser Functions here, or we could create a whole new section for them next to "User Interfaces".